webapp/src/modules/vendor/decentraland/favorites/api.ts

```diff
@@ -11,6 +11,27 @@ export const MARKETPLACE_FAVORITES_SERVER_URL = config.get(
 )!
 
 class FavoritesAPI extends BaseAPI {
+  async getWhoFavoritedAnItem(
+    itemId: string,
+    limit: number,
+    offset: number
+  ): Promise<{ addresses: string[]; total: number }> {
+    const response = await fetch(
+      `${MARKETPLACE_FAVORITES_SERVER_URL}/picks/${itemId}?limit=${limit}&offset=${offset}`
+    )
+
+    const parsedResponse = await response.json()
+
+    if (!response.ok) {
+      throw new Error(parsedResponse?.message ?? 'Unknown error')
```

Comment:
I think we can use the translation module here (after importing the function `t`):
```suggestion
      throw new Error(parsedResponse?.message ?? t('global.unknown_error'))
```

Reply:
Done!